user_guide_src/source/changelogs/v4.3.0.rst

```diff
@@ -220,7 +220,9 @@ Helpers and Functions
 Error Handling
 ==============
 
-- You can now log deprecation errors instead of throwing them. See :ref:`logging_deprecation_errors` for details.
+- You can now log deprecation warnings instead of throwing them. See :ref:`logging_deprecation_warnings` for details.
```

Comment:
Does *throwing deprecation warnings* make sense?
```suggestion
- You can now log deprecation warnings instead of throwing exceptions. See :ref:`logging_deprecation_warnings` for details.
```